src/utils/table.ts

```diff
@@ -12,36 +24,36 @@ import { Table } from "types/Table";
  * @param titlePrefix what should the tables be called (will have the table number appended to it)
  * @param appendTableNumber whether to append the table number to the title or not
  * @param startFrom what number should we start from when generating table numbers in the title
+ * @param subtitle what should the tables subtitle be
  */
-export const generateTables: (props: {
-  num: number;
-  capacity: number;
-  rows?: number;
-  columns?: number;
-  titlePrefix?: string;
-  appendTableNumber?: boolean;
-  startFrom?: number;
-}) => Table[] = ({
-  num,
-  capacity,
-  rows = DEFAULT_TABLE_ROWS,
-  columns = DEFAULT_TABLE_COLUMNS,
-  titlePrefix = "Table",
-  appendTableNumber = true,
-  startFrom = 1,
-}) =>
-  Array.from(Array(num)).map((_, idx) => {
+export const generateTables = (props: GenerateTablesProps): Table[] => {
+  const {
+    num,
+    capacity,
+    rows = DEFAULT_TABLE_ROWS,
+    columns = DEFAULT_TABLE_COLUMNS,
+    titlePrefix = "Table",
+    appendTableNumber = true,
+    startFrom = 1,
+    subtitle,
```

Comment:
I've just tried to run the script with the default settings (specifying just `num` and `capacity`) and looks like we need to add a default value for `subtitle`.
Context: 
```
Transaction failed:  Error: Update() requires either a single JavaScript object or an alternating list of field/value pairs that can be followed by an optional precondition. Value for argument "dataOrField" is not a valid Firestore value. 
  Cannot use "undefined" as a Firestore value (found in field "`config.tables`.`0`.subtitle"). If you want to ignore undefined values, enable `ignoreUndefinedProperties`.
    at WriteBatch.update (/home/viktoryia/work/sparkle/node_modules/@google-cloud/firestore/build/src/write-batch.js:380:23)
    at Transaction.update (/home/viktoryia/work/sparkle/node_modules/@google-cloud/firestore/build/src/transaction.js:222:26)
    at /home/viktoryia/work/sparkle/scripts/upload-table-config.ts:85:22
    at step (/home/viktoryia/work/sparkle/scripts/upload-table-config.ts:33:23)
    at Object.next (/home/viktoryia/work/sparkle/scripts/upload-table-config.ts:14:53)
    at fulfilled (/home/viktoryia/work/sparkle/scripts/upload-table-config.ts:5:58)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)

```